client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -479,7 +479,7 @@ public int pushOrMergeData(
             mapId,
             attemptId,
             partitionId,
-            0,
+            -1,
```

Comment:
In fact it should never reach here after this pr, I thought to add some log but it's in short-circuit condition so I didn't.